package.json

```diff
@@ -26,11 +26,11 @@
   "dependencies": {
     "@actions/core": "1.2.4",
     "@actions/exec": "1.0.4",
+    "@iarna/toml": "^2.2.5",
     "conventional-changelog": "3.1.21",
     "conventional-recommended-bump": "6.0.9",
     "git-semver-tags": "4.0.0",
     "object-path": "^0.11.4",
-    "toml": "^3.0.0",
```

Comment:
There a reason for the switch?

Reply:
yes
this dependency has no `stringify`
this is used if the version can't be read from the file and we need to write a valid toml file